assets/app/views/projects.html

```diff
@@ -4,11 +4,14 @@
     <h1 style="display: inline-block;">Projects</h1>
     <a ng-if="canCreate" href="createProject" style="margin-top: 10px;" class="btn btn-lg btn-primary pull-right">New Project</a>
     <alerts alerts="alerts"></alerts>
-    <div ng-repeat="project in projects | orderByDisplayName">
-      <div row flex cross-axis="center" class="tile tile-project tile-click">
+    <div ng-repeat="project in (items = (projects | orderByDisplayName)) ">
+       <div row flex cross-axis="center" class="tile tile-project tile-click">
         <div flex>
           <h2 class="project">
-            <a class="tile-target" href="project/{{project.metadata.name}}">{{(project | displayName) || project.metadata.name}}</a>
+            <a class="tile-target" href="project/{{project.metadata.name}}">{{(project | displayName) || project.metadata.name}}
```

Comment:
Same thing, too much logic in the view. Build displayName:count in the controller and check against that here. No need to || with metadata name, display name filter falls back to that now